Mathlib/Algebra/Order/Ring/WithTop.lean

```diff
@@ -3,7 +3,7 @@ Copyright (c) 2016 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad, Leonardo de Moura, Mario Carneiro
 ! This file was ported from Lean 3 source module algebra.order.ring.with_top
-! leanprover-community/mathlib commit 550b58538991c8977703fdeb7c9d51a5aa27df11
+! leanprover-community/mathlib commit e7e2ba8aa216a5833b5ed85a93317263711a36b5
```

Comment:
This one (#1508) seems to be a manual translation that doesn't match the mathport output

https://github.com/leanprover-community/mathlib3port/blob/635c065bc4d63286524c51fc8dcbbea9c7da33c2/Mathbin/Algebra/Order/Ring/WithTop.lean#LL424-L436C43

Reply:
It seems to me that it does match the mathport output and is correct  (modulo simple style-fixes) .

Note that the mathport output had a lot of `tactic; tactic₂` which have been fixed to
```
tactic
tactic₂
```
and therefore they might not look alike at first glance.